src/main/java/controller/UserController.java

```diff
@@ -116,4 +123,12 @@ public ResponseEntity updateUser(@RequestBody User user) {
         }
     }
 
+    private ResponseEntity createResponse(User u) {
```

Comment:
Good idea. Although I think we should move this out to the Abstract controller layer so we can reuse in every controller.